src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/AncestralNamespaceContextManager.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Xml;
+using System.IO;
+using System.Text;
+using System.Collections;
+
+namespace System.Security.Cryptography.Xml
+{
+    abstract internal class AncestralNamespaceContextManager
```

Comment:
The using statements? Unfortunately no. Doesn't remove unneeded ones either. It will need to be a manual step unless someone writes a Roslyn script